relay-server/src/services/buffer/stack_provider/sqlite.rs

```diff
@@ -1,36 +1,65 @@
-use relay_config::Config;
 use std::error::Error;
 
+use relay_config::Config;
+
 use crate::services::buffer::common::ProjectKeyPair;
 use crate::services::buffer::envelope_store::sqlite::{
     SqliteEnvelopeStore, SqliteEnvelopeStoreError,
 };
 use crate::services::buffer::stack_provider::{
     InitializationState, StackCreationType, StackProvider,
 };
-use crate::SqliteEnvelopeStack;
+use crate::statsd::RelayTimers;
+use crate::{Envelope, EnvelopeStack, SqliteEnvelopeStack};
+
+/// Maximum number of envelopes that are inserted into the database during draining.
+pub const DEFAULT_DRAIN_BATCH_SIZE: usize = 100;
```

Comment:
Why not use the configured batch size? If the configured batch size is the optimal one, we should use it for both regular spooling and flushing IMO.